scripts/run_lighthouse_tests.py

```diff
@@ -69,16 +69,38 @@
 _PARSER.add_argument(
     '--shard', help='Sets the shard for the lighthouse tests',
     required=True, choices=['1', '2'])
+
 _PARSER.add_argument(
     '--skip_build', help='Sets whether to skip webpack build',
     action='store_true')
 
+_PARSER.add_argument(
+    '--record_screen', help='Sets whether LHCI Puppeteer script is recorded',
+    action='store_true')
+
+
+def run_lighthouse_puppeteer_script(record: bool=False) -> None:
```

Comment:
```suggestion
def run_lighthouse_puppeteer_script(record: bool = False) -> None:
```

Normally we omit spaces when specifying arguments, but not when we're also setting a type hint

Reply:
Added spacing for type hint in new commit.